autode/neb/original.py

```diff
@@ -609,23 +677,15 @@ def calculate(
         plt.close()
         return None
 
-    def get_species_saddle_point(self) -> "autode.species.Species":
-        """Find a TS guess species for this NEB: highest energy saddle point"""
-        if self.images.peak_idx is None:
+    @property
+    def peak_species(self) -> Optional[Species]:
```

Comment:
The examples in the doc also need to be updated I believe (the pages for transition states and for NEB/CINEB)

Reply:
good spot – cheers